plexpy/config.py

```diff
@@ -66,6 +66,7 @@ def bool_int(value):
     'PMS_UPDATE_DISTRO': (str, 'PMS', ''),
     'PMS_UPDATE_DISTRO_BUILD': (str, 'PMS', ''),
     'PMS_WEB_URL': (str, 'PMS', 'https://app.plex.tv/desktop'),
+    'PMS_UPDATE_NOTIFY_INTERVAL': (int, 'Advanced', 24),
```

Comment:
This is bugging me that it should be above `PMS_WEB_URL` with the other `PMS_UPDATE_*` settings.

```suggestion
    'PMS_UPDATE_CHECK_INTERVAL': (int, 'Advanced', 24),
```